modules/s3-api-compatible-services-configuring-acs.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-s3-api-compatible-services.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="s3-api-compatible-services-configuring-acs_{context}"]
+= Configuring S3 API compatible integrations in {product-title}
+
+To configure S3 API compatible backups, create a new integration in {product-title}.
+
+.Prerequisites
+* An existing S3 Bucket. +
+To create a new bucket with required permissions, see your S3 provider documentation.
+* `Read`, `write`, and `delete` permissions for the S3 bucket, the *Access key ID*, and the *Secret access key*.
+
+.Procedure
+. In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *External backups* section and select *S3 API Compatible*.
+. Click *New Integration*.
+. Enter a name for *Integration Name*.
+. Enter the number of backups to retain in the *Backups To Retain* box.
+. For *Schedule*, select the backup frequency as daily or weekly as well as the time to run the backup process.
```

Comment:
```suggestion
. For *Schedule*, select the backup frequency as daily or weekly, and select the time to run the backup process.
```
Not sure if you select the time or enter it since I don't have access to the GUI, so use whichever is appropriate.

Reply:
You select the time.